lib/gateways/snapshot-gateway.test.js

```diff
@@ -344,6 +363,21 @@ describe('SnapshotGateway', () => {
       expect(result).toBe(snapshot);
     });
 
+    it('removes the ttl on save', async () => {
+      const snapshot = new Snapshot({
+        id: 1,
+        assets: ['an asset'],
+        notes: '',
+        vulnerabilities: ['a vulnerability']
+      });
+      snapshot.ttl = 123;
+      const snapshotGateway = new SnapshotGateway({ client, tableName });
+
+      const result = await snapshotGateway.save({ snapshot });
+
+      expect(result.ttl).toBeUndefined();
+    });
```

Comment:
It also passes with assets and vulnerabilities == [].
Is the save possible if no options are selected?

Reply:
v. good point

Reply:
Ah, I guess it's been mentioned already